tests/core/parser/PDFParser.spec.ts

```diff
@@ -99,6 +99,24 @@ describe(`PDFParser`, () => {
     expect(object).toBeInstanceOf(PDFInvalidObject);
   });
 
+  it(`throws an error with invalid indirect objects when throwOnInvalidObject=true`, (done) => {
```

Comment:
This test can be refactored to use `expect(...).rejects.toBeInstanceOf(Error)`. See previous comment.